nextflow_schema.json

```diff
@@ -0,0 +1,498 @@
+                },
+                "skip_spadeshybrid": {
+                    "type": "boolean",
+                    "description": "Skip SPAdes hybrid assembly (only available when using manifest input)."
+                },
+                "skip_megahit": {
+                    "type": "boolean",
+                    "description": "Skip MEGAHIT assembly."
+                },
+                "skip_quast": {
+                    "type": "boolean",
+                    "description": "Skip metaQUAST."
+                }
+            }
+        },
+        "binning_options": {
+            "title": "Binning options",
+            "type": "object",
+            "description": "",
+            "default": "",
+            "properties": {
+                "skip_binning": {
+                    "type": "boolean",
+                    "description": "Skip metagenome binning."
+                },
+                "min_contig_size": {
+                    "type": "integer",
+                    "default": 1500,
+                    "description": "Minimum contig size to be considered for binning and for bin quality check.",
+                    "help_text": "For forwarding into downstream analysis, i.e. QUAST and BUSCO, and reporting."
+                },
+                "min_length_unbinned_contigs": {
+                    "type": "integer",
+                    "default": 1000000,
+                    "description": "Minimal length of contigs that are not part of any bin but treated as individual genome.",
+                    "help_text": "Contigs that do not fulfill the thresholds of `--min_length_unbinned_contigs` and `--max_unbinned_contigs` are pooled for downstream analysis and reporting, except contigs that also do not fullfill `--min_contig_size` are not considered further."
+                },
+                "max_unbinned_contigs": {
+                    "type": "integer",
+                    "default": 100,
+                    "description": "Maximal number of contigs that are not part of any bin but treated as individual genome.",
+                    "help_text": "Contigs that do not fulfill the thresholds of `--min_length_unbinned_contigs` and `--max_unbinned_contigs` are pooled for downstream analysis and reporting, except contigs that also do not fullfill `--min_contig_size` are not considered further."
+                }
+            }
+        },
+        "bin_quality_check_options": {
+            "title": "Bin quality check options",
+            "type": "object",
+            "description": "",
+            "default": "",
+            "properties": {
+                "skip_busco": {
+                    "type": "boolean",
+                    "description": "Disable bin QC with BUSCO."
+                },
+                "busco_reference": {
+                    "type": "string",
+                    "default": "https://busco-data.ezlab.org/v4/data/lineages/bacteria_odb10.2020-03-06.tar.gz",
```

Comment:
https://nf-co.re/mag/usage#--busco_reference lists (wrongly) https://busco.ezlab.org/datasets/bacteria_odb9.tar.gz as default, here it is correct, not sure why. Might not be a problem, just want to make sure.

Reply:
this is resolved, my bad.

Reply:
https://nf-co.re/mag/usage#--busco_reference also shows the last release version. However, for some reason https://nf-co.re/mag/dev/usage#bin-quality-check-options doesn't show the default